apps/docs/docs/state/actions/actions.mdx

```diff
@@ -10,18 +10,17 @@ hide_title: true
 [![npm](https://img.shields.io/npm/v/%40rx-angular%2Fstate.svg)](https://www.npmjs.com/package/%40rx-angular%2Fstate)
 ![rx-angular CI](https://github.com/rx-angular/rx-angular/workflows/rx-angular%20CI/badge.svg?branch=master)
 
-> This package provides a quick way to create an action channel and some configuration options.
+> This package provides a handy way handle have reactive events of any type.
+> It fit's perfectly for situations like debounce user input or switchMap a HTTP request
```

Comment:
```suggestion
> `RxActions` is a powerful yet simple tool to manage actions of any type.
> It can be seen as the glue between user based events and your applications state. 
```